dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -154,8 +160,40 @@ public void testTargettedRefresh() throws ApiException {
         assertEquals("should find 4 valid versions for bitbucket workflow, found : " + refreshBitbucket.getWorkflowVersions().stream()
                 .filter(WorkflowVersion::isValid).count(), 4,
             refreshBitbucket.getWorkflowVersions().stream().filter(WorkflowVersion::isValid).count());
+    }
+
+    @Test
+    public void testHostedDelete() {
+        final ApiClient webClient = getWebClient();
+        WorkflowsApi workflowApi = new WorkflowsApi(webClient);
+        UsersApi usersApi = new UsersApi(webClient);
+        User user = usersApi.getUser();
+        usersApi.refreshWorkflows(user.getId());
+        // do targetted refresh, should promote workflow to fully-fleshed out workflow
+        final Workflow workflowByPathGithub = workflowApi.getWorkflowByPath(DOCKSTORE_TEST_USER2_HELLO_DOCKSTORE_WORKFLOW);
+        final Workflow refreshGithub = workflowApi.refresh(workflowByPathGithub.getId());
```

Comment:
I'm a bit confused for what this test is for. I thought a hosted workflow would not be connected to github.

Reply:
Both of these are actually expecting failures. Simply put, many of our existing endpoints like refresh, refresh by org, etc. should just skip over hosted workflows. We don't have tests for those yet (but should). Conversely, this is making sure that the new endpoints can't break the old content. i.e. you can't use the new CRUD endpoints to create or delete revisions of our old workflows harvested from GitHub